Makefile

```diff
@@ -1,5 +1,10 @@
+UNVERSIONED = unversioned
```

Comment:
I think I'd get rid of this and just hardcode it as the default. It's not used anywhere outside of `version` is it?

Reply:
It is used in lines 66 and 73...
`ifeq ($(version), ${UNVERSIONED})`
Billy added this bit. I guess so it can be amended by a dev for local development, as required.
I'm wondering if having this version code and the conditions around tagging and pushing will confuse matters - as we'd not want this pushed from a Makefile anyway (only from Concourse). Maybe worth removing entirely?
PS - this Makefile isn't used at all in the pipeline.

Reply:
Removed the ${UNVERSIONED} variable. The conditions using this have now also been removed (as per https://github.com/companieshouse/test-data-generator/pull/55#discussion_r386300916) so not required.